core/rfb.js

```diff
@@ -195,7 +195,7 @@ export default class RFB extends EventTargetMixin {
             }
         });
         this._sock.on('close', (e) => {
-            Log.Debug("WebSocket on-close event");
+            Log.Debug("WebChannel on-close event");
```

Comment:
These are user visible and I'd like to avoid exporting our new internal "WebChannel" term. Perhaps these should have something more generic, like "Connection close event"?